backend/server.js

```diff
@@ -21,21 +21,23 @@ const allowedOrigins = [
     /https:\/\/deploy-preview-\d+--delightful-daifuku-a9f6ea\.netlify\.app/,
 ];
 
-app.use(
-    cors({
-        origin: function (origin, callback) {
-            // Allow requests with no `origin` (like from Postman or server-side scripts)
-            if (!origin || allowedOrigins.some((o) =>
-                typeof o === "string" ? o === origin : o.test(origin)
-            )) {
-                callback(null, true);
-            } else {
-                // Provide a more informative error message if necessary
-                callback(new Error("CORS policy: This origin is not allowed."));
-            }
-        },
-    })
-);
+// app.use(
+//     cors({
+//         origin: function (origin, callback) {
+//             // Allow requests with no `origin` (like from Postman or server-side scripts)
+//             if (!origin || allowedOrigins.some((o) =>
+//                 typeof o === "string" ? o === origin : o.test(origin)
+//             )) {
+//                 callback(null, true);
+//             } else {
+//                 // Provide a more informative error message if necessary
+//                 callback(new Error("CORS policy: This origin is not allowed."));
+//             }
+//         },
+//     })
+// );
+
```

Comment:
Same comment problem as mention above

Reply:
@AlfiyaSiddique, I commented this code because of cors setup for local machine.